st2common/tests/unit/test_operators.py

```diff
@@ -277,3 +277,13 @@ def test_nexists(self):
         self.assertFalse(op(1, None), 'Should return False')
         self.assertFalse(op('foo', None), 'Should return False')
         self.assertTrue(op(None, None), 'Should return True')
+
+    def test_inside(self):
```

Comment:
Maybe testing for passing None as criteria_pattern can be added here, if sensible...

Reply:
None type tests added.